k8s/with-defaults.bzl

```diff
@@ -97,6 +101,7 @@ k8s_defaults = repository_rule(
         "user": attr.string(mandatory = False),
         "namespace": attr.string(mandatory = False),
         "image_chroot": attr.string(mandatory = False),
+        "kubectl_args": attr.string_list(mandatory = False),
```

Comment:
not sure if this should be a `string_list` or a `string`

Reply:
a string_list, imo